### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by meitangdehulu**

### CODE REVIEW

File: src/main/resources/assets/mekanism/lang/zh_cn.lang [hunk 1]
```diff
@@ -220,7 +220,7 @@ tile.EnergyCubeElite.name=精英能量立方
 tile.EnergyCubeUltimate.name=终极能量立方
 tile.EnergyCubeCreative.name=创造能量立方
 
-//Factory Installers
+//Energy Cubes
```
Comment (thiakil) [inline src/main/resources/assets/mekanism/lang/zh_cn.lang]: This doesn't make sense, is it an error in the base english file?

**Comment by meitangdehulu** [comment]
Yes, I'll fix it back. 
@thiakil

**Comment by thiakil** [comment]
Please include these new translations, and I think it's good to go. Nice work :)

https://github.com/aidancbrady/Mekanism/commit/2f8f71862c2167ba7f3c09d10d396d23fbb7d494#diff-cbf6ee77a88a174288939fc6fac40851R724

**Comment by thiakil** [comment]
Looks good to me, thanks for your efforts!

**Comment by thiakil** [comment]
Could you please verify that this merged properly? I messed it up.

https://github.com/aidancbrady/Mekanism/commit/72540ce978435dc817a51fe9245ae8ef388242c2

**Pull request merged**